mender-convert-modify

```diff
@@ -261,6 +261,17 @@ if [ "${MENDER_DATA_PART_GROWFS}" == "y" ]; then
   MENDER_DATA_PART_FSTAB_OPTS="${MENDER_DATA_PART_FSTAB_OPTS},x-systemd.growfs"
 fi
 
+if echo ${MENDER_ROOT_PART_FSTAB_OPTS} | tr ',' '\n' | egrep -q "^ro$"; then
+  if ! echo "${MENDER_ROOT_PART_MKFS_OPTS}" | fgrep -q -- "-O ^64bit"; then
+    log_warn "EXT4 64bits feature is known to create unstable checksums on read-only file systems, add \"-O ^64bit\" to MENDER_ROOT_PART_MKFS_OPTS to remove it"
+  fi
+  if ! echo "${MENDER_ROOT_PART_MKFS_OPTS}" | fgrep -q -- "-O ^has_journal"; then
+    log_warn "EXT4 journal feature is known to create unstable checksums on read-only file systems, add \"-O ^has_journal\" to MENDER_ROOT_PART_MKFS_OPTS to remove it"
+  fi
+  if [ "${MENDER_ROOT_PART_FS_PASSNO}" != "0" ]; then
+    log_warn "fsck enabled on a read-only file system, set MENDER_ROOT_PART_FS_PASSNO to 0 to disable it "
```

Comment:
No strong opinion, but I think I would vote for `log_info` level here. I don't consider it wrong to use fsck on a read-only filesystem. This is still useful in cases where hardware storage errors have occurred, or when the filesystem has become read-write mounted because of some manual action.

Reply:
Then I will remove it (I don't think is a point to "log" a recommendation) and rephrase [comment here](https://github.com/mendersoftware/mender-convert/blob/master/configs/mender_convert_config#L41).